topo/node/ceos/ceos.go

```diff
@@ -49,9 +52,16 @@ type Node struct {
 
 // Add validations for interfaces the node provides
 var (
-	_ node.Certer       = (*Node)(nil)
```

Comment:
Please keep the Certer implementation but have it return an [unimplemented error](https://pkg.go.dev/google.golang.org/grpc@v1.48.0/codes#pkg-types) with an adequate error message stating exactly how a user should create the certs

so if a user tries to create new self signed certs for the node using the kne_cli they can be pointed to the right way to do it (send a request to the controller / certs must be included in the topo textproto at startup... etc.)